server/bundles/io.cloudbeaver.server/schema/service.navigator.graphqls

```diff
@@ -59,7 +59,9 @@ type NavigatorNodeInfo {
     # Node human readable name
     name: String
     #Node full name
-    fullName: String
+    fullName: String @deprecated
+    #Node plain name
+    plainName: String
```

Comment:
https://github.com/dbeaver/pro/wiki/GraphQL-Schema-Guidelines
please specify deprecation message: version when deprecated, comment what to use instead

Reply:
@Wroud could you explain what is used instead of fullName?

Reply:
i don't know but if field deprecated you need to describe in favor of what
i suppose it's `name`